### DISCUSSION TITLE
Options + ] command is not working on mac

### DISCUSSION THREAD

**Discussion by collins-anaba** [community/community#6790] [Copilot Conversations] [answered]
Has anyone have issue with the option + ] on their mac?
when i create a function on vs code, the copilot creates a solution for me but when i enter option + ] command to see another solution , it doesn't execute. is there something i'm doing wrong?

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Sometimes, Copilot just doesn't have any unique alternatives for the suggestion.

But if you can't get any alternatives at all... Can you please check if there's no other extension that overrides the keyboard shortcut? You can do that by going to Code -> Preferences -> Keyboard Shortcuts and then searching for `"alt+]"`.

That's what I see in this case:
![image](https://user-images.githubusercontent.com/5427083/139044400-1548351d-e001-4322-ab2d-0215d2d42dbc.png)

**Reply by collins-anaba** [reply]
<img width="670" alt="Screen Shot 2021-10-27 at 10 50 58" src="https://user-images.githubusercontent.com/39745373/139101487-46c52958-ac31-4d4f-844c-154024a9e81e.png">

mine is showing the same thing. You could be right in the fact it only has one suggestion to offer

**Reply by ivanistheone** [reply]
> Sometimes, Copilot just doesn't have any unique alternatives for the suggestion.

I ran into this too. It's a bit surprizing, but in certain contexts there is only one suggestion offered.

**Comment by swfjoe** [comment]
I'm having the same problem in IntelliJ. I've checked the preferences and system keyboard shortcuts, but I all get are single or double quotes when I try these shortcuts. “‘“‘“‘“‘

**Reply by swfjoe** [reply]
I saw it in another post - this only works for Java (confirmed), and not JavaScript.

**Reply by giansegato** [reply]
Agree, this doesn't work on PyCharm too.

**Comment by jmcerrejon** [comment]
I think the problem is the keyboard layout. I changed to English (from Spanish) and it works like a charm. ~The fact is... Why can't we set that keys on keyboard shortcuts settings?~

**FIX:** You can set the next and fix that with the keys: `editor.action.inlineSuggest.showPrevious editor.action.inlineSuggest.showNext`

**Reply by vinicius-mariano** [reply]
This fix worked for me. Thanks! 🚀

**Comment by CarloCattano** [comment]
Have the same on vscode with a windows machine. The] has no effect, will follow the tips here for next try

**Comment by CarloCattano** [comment]
So if you use Spanish qwerty is at alt+¡. Hovering over a solution gracefully suggests it and shows current binding. Bravo

**Comment by AurethiaVeritas** [comment]
I'm having the same issue in IntelliJ. it seems like on macOS when you press Option key the system will change the keymap so the ] or [ key will no longer be themself

**Reply by n2nco** [reply]
<img width="587" alt="image" src="https://user-images.githubusercontent.com/36323943/177381173-59c35c47-9e81-467b-ad84-1bfd1b22562e.png">

I don't ever get 'next' suggestions.. no matter what has been typed

**Reply by n2nco** [reply]
Yet when I 'Open Github Copilot' it synthesizes many solutions e.g.
<img width="1309" alt="image" src="https://user-images.githubusercontent.com/36323943/177381673-4e71dd70-f4a7-400a-a739-f0d31380b389.png">

**Comment by CaliforniaDev** [comment]
Got into the keyboard shortcuts settings. search for alt + ] or alt + [  

You'll see the keybinding for you systems default, and the other for GitHub Copilot. Just right click the GitHub Copilot key binding and then is should work perfectly fine.
